docs/edgeql/funcops/datetime.rst

```diff
@@ -299,82 +296,6 @@ Date and Time
 ----------
 
 
-.. eql:function:: std::duration_get(dt: duration, el: str) -> float64
```

Comment:
We should maybe add a function or a cast to bigint returning the number of microseconds in the duration.

Reply:
We don't need bigint here, because internally there's only int64 microseconds.